### PR TITLE
Docs  update video v2

### PR DESCRIPTION
**Pull request by oliviaflory**
### Related Ticket(s)

#1624 

### Description

Update video docs

### Changelog

**New**

- {{new thing}}

**Changed**

- description

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/components/content-block-cards.mdx [hunk 1]
```diff
@@ -25,8 +25,9 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
-By default the Content block cards simply supports only text for moments needing richer content, and sets proper
-expectations for the user to make their next journey decision from within the page's narrative.
+The default version pairs the Content block heading and [card group](https://www.ibm.com/standards/carbon/components/card-group) component. Use when you need to introduce a card group as its own content block without a description.
```
Comment (IgnacioBecerra) [inline src/pages/components/content-block-cards.mdx:28]: Small question regarding capitalization here -- in `Content block` the `c` is capitalized, but not the `c` in `card group`. Should only the first component be capitalized in a sentence?

File: src/pages/components/feature-section.mdx [hunk 2]
```diff
@@ -28,7 +28,7 @@ Additional usage guidelines will be available soon.
 
 ## Overview
 
-Feature section is a full-width section, designed to create a visual break from the rest of the page and focus the viewer's attention to one single story. It contains expressive imagery and a CTA linking people to more details.
+Feature section is a full-width section designed to create a visual break from the rest of the page and focus the viewer's attention on a single story. It contains expressive imagery and a CTA linking people to more details. Feature section is meant to be used sparingly to draw attention – avoid stacking multiple Feature sections in a row or including too many on a single page.
```
Comment (IgnacioBecerra) [inline src/pages/components/feature-section.mdx:31]: Likewise here, the second `Feature section` is capitalized too.

Comment (oliviaflory) [reply src/pages/components/feature-section.mdx:31]: aw man good catch, we are trying not to capitalize any component names unless they are in the middle of the sentence, but we keep messing that up lol

File: src/pages/components/content-block-cards.mdx (hunk 1 above)
Comment (oliviaflory) [reply src/pages/components/content-block-cards.mdx:28]: Oh man @IgnacioBecerra idk how I did this, but this PR should have been only the video page!

**Comment by oliviaflory** [comment]
Closing due to too many conflicts, will open a new PR

**Pull request closed without merging**